src/paperqa/docs.py

```diff
@@ -781,26 +781,53 @@ async def aquery(  # noqa: PLR0912
             # Only keep "\nFrom {citation}" if we are showing detailed citations
             context_inner_prompt = context_inner_prompt.replace("\nFrom {citation}", "")
 
-        inner_context_strs = [
-            context_inner_prompt.format(
-                name=c.id,
-                text=c.context,
-                citation=c.text.doc.formatted_citation,
-                **(c.model_extra or {}),
-            )
-            for c in filtered_contexts
-        ]
+        context_str_body = ""
+        if answer_config.group_contexts_by_question:
+            contexts_by_question: dict[str, list] = {}
+            for c in filtered_contexts:
+                # Fallback to the main session question if not available.
```

Comment:
Can you document why would the question not be available? Is it an "extra" of the model?